Vanilla-JS-Projects/Basic/Electronic-Drum-Kit/README.md

```diff
@@ -0,0 +1,94 @@
+<h1 align='center'><b>💥 Electronic Drum Kit 💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+<ul align="left">
+    <li><img src="https://img.shields.io/badge/HTML5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white" alt="HTML5 Badge"></li>
+    <li><img src="https://img.shields.io/badge/CSS3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white" alt="CSS3 Badge"></li>
+    <li><img src="https://img.shields.io/badge/JavaScript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E" alt="JavaScript Badge"></li>
+</ul>

```

Comment:
Don't create this list.
![image](https://github.com/user-attachments/assets/4b1ffb1b-ad86-4cbd-a2e9-0477fe9b4e40)

You have to un-comment those tech stacks which are necessary for this project

Reply:
Do you mean just remove the pictures and add them as text?

Reply:
No, in readme there are some of tech stacks links are shown which are commentted. You have to remove those tech stack comments which are necessary to show for your project. As here only HTML, CSS and JS

Reply:
done check it out